pkgs/tools/networking/dd-agent/integrations-core.nix

```diff
@@ -33,7 +33,7 @@
 #
 # [1]: https://github.com/DataDog/integrations-core
 
-{ pkgs, python, extraIntegrations ? {} }:
+{ pkgs, python, extraIntegrations ? { }, }:
```

Comment:
Why are we passing `pkgs` here? We should import lib, fetchFromGitHub and others as needed. Current implementation makes it less overridable.

Reply:
it is a little unusual. i was thinking about refactoring the integrations but didn't end up figuring out a way to improve them.